packages/kubecost/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,10 +1,18 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -4,3 +4,7 @@
+@@ -1,10 +1,14 @@
+ annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: kubecost
```

Comment:
The `catalog.cattle.io/display-name` annotation for what you want the UI to display is missing (more info on it [here](https://github.com/rancher/partner-charts/tree/main-source#4-make-changes)). Make sure you use the make commands and follow the workflow order to make the change, and re-run make charts again. You need to re-run make charts every time you make changes, and it should replace your previous make charts commit.

Reply:
Thanks! I've added the display name in commit #3 and re-ran `make charts` in #4.